package.json

```diff
@@ -12,13 +12,13 @@
     "url": "git+https://github.com/seek-oss/vocab.git"
   },
   "scripts": {
-    "start.direct": "yarn dev && manypkg run fixture-direct compile && yarn start-fixture fixture-direct",
-    "start.server": "yarn dev && manypkg run fixture-server compile && yarn start-fixture fixture-server",
-    "start.simple": "yarn dev && manypkg run fixture-simple compile && yarn start-fixture fixture-simple",
+    "start.direct": "yarn dev && manypkg run @fixtures/direct compile && yarn start-fixture direct",
```

Comment:
> `manypkg run @fixtures/direct`

FYI pnpm has this feature (plus more) built-in: https://pnpm.io/filtering#--filter-package_name

Reply:
`manypkg` is too aggro for my taste so I'm really keen to move away from it. The biggest offence is not supporting `workspace:*` which leads to all sorts of bugs. For example:
- in Braid `site` depends on `braid-design-system` with `*` (not `workspace:*`) because that's what `manypkg` considers the _correct_ way
- there's a newer version of `braid-design-system` and you're on a branch that is behind master, running `pnpm install` will pull in the latest `braid-design-system`; this happened many times while working on compiled Braid

Reply:
Yeah it has been a pain, wanting to move away from it too for anything other than just keeping package versions in line. Will definitely swap to `pnpm --filter`.